src/pages/billing/Project.js

```diff
@@ -258,32 +335,27 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
     setProjectUsers(projectUsers)
   })
 
-
   // Lifecycle
-  Utils.useOnMount(() => {
-    refresh()
-    loadBillingInfo()
-  })
+  Utils.useOnMount(refresh)
 
-  useEffect(() => {
-    StateHistory.update({ projectUsers })
-  }, [projectUsers])
+  useEffect(() => StateHistory.update({ projectUsers }), [projectUsers])
```

Comment:
See above
```suggestion
  useEffect(() => { StateHistory.update({ projectUsers }), [projectUsers] })
```

Reply:
My bad, suggestion should have been:
```suggestion
  useEffect(() => { StateHistory.update({ projectUsers }) }, [projectUsers])
```